src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1515,6 +1517,45 @@ public Egg get(PlayerEggThrowEvent event) {
 			}
 		}, EventValues.TIME_NOW);
 
+		// BrewEvent
+		EventValues.registerEventValue(BrewEvent.class, Block.class, new Getter<Block, BrewEvent>() {
+			@Override
+			@Nullable
+			public Block get(BrewEvent event) {
+				return event.getBlock();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(BrewEvent.class, Location.class, new Getter<Location, BrewEvent>() {
+			@Override
+			@Nullable
+			public Location get(BrewEvent event) {
+				return event.getBlock().getLocation();
+			}
+		}, EventValues.TIME_NOW);
```

Comment:
You don't have to register the location as an event value if it's only grabbing the block location. Skript will convert the block to a location. Only register a location event value if the event location is different than the block location.